google/cloud/google_cloud_cpp_common.bzl

```diff
@@ -26,6 +26,7 @@ google_cloud_cpp_common_hdrs = [
     "iam_binding.h",
     "iam_bindings.h",
     "iam_policy.h",
+    "iam_updater.h",
```

Comment:
This seems like the wrong place for these headers, since they seem to depend on gRPC and this list is for the "common" deps, which shouldn't have a dep on grpc. I suspect the other iam-related headers that already existed here may be in the wrong place too.

Reply:
`iam_updater.h` moved in https://github.com/googleapis/google-cloud-cpp/pull/7287.